prow/config.yaml

```diff
@@ -139,10 +139,9 @@ presubmits:
     trigger: "@k8s-bot (verify )?test this"
 
   - name: pull-kubernetes-e2e-gce
-    always_run: true
-    context: Jenkins GCE e2e
-    rerun_command: "@k8s-bot cvm gce e2e test this"
-    trigger: "@k8s-bot (cvm )?(gce )?(e2e )?test this"
+    context: pull-kubernetes-e2e-gce
```

Comment:
I would rather *only* remove the `always_run` lines in this PR. We can change contexts over when we have a context-changing script ready.